pkg/maps/eppolicymap/eppolicymap.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2016-2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package eppolicymap
+
+import (
+	"fmt"
+	"net"
+	"sync"
+	"unsafe"
+
+	"github.com/cilium/cilium/pkg/bpf"
+	"github.com/cilium/cilium/pkg/logging"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+	"github.com/cilium/cilium/pkg/maps/lxcmap"
+	"github.com/cilium/cilium/pkg/maps/policymap"
+	"github.com/cilium/cilium/pkg/option"
+	"github.com/spf13/viper"
+)
+
+var log = logging.DefaultLogger.WithField(logfields.LogSubsys, "map-ep-policy")
+
+const (
+	mapName = "cilium_ep_to_policy"
+
+	// MaxEntries represents the maximum number of endpoints in the map
+	MaxEntries = 65535
+)
+
+type EndpointKey struct{ bpf.EndpointKey }
```

Comment:
I looks like this type can be unexported

Reply:
sure.